src/styles/utilis/_variables.scss

```diff
@@ -0,0 +1,15 @@
+$color-blue:#00acdc;
+$color-white: #fff;
```

Comment:
if it's used for background - please add background to variable name

Reply:
This color is given to many elements (screen), not just the background, so, for the sake of code readability, I would prefer to leave it as is. 
![red](https://github.com/mate-academy/layout_catalog/assets/131079210/cb9480e3-02cb-4e62-b140-e094a42089fd)